components/contribute-cards/ContributeCrypto.js

```diff
@@ -20,7 +20,7 @@ const messages = defineMessages({
 const ContributeCrypto = ({ intl, collective, contributors, stats, ...props }) => {
   return (
     <Contribute
-      route={`/${collective.slug}/contribute/crypto`}
+      route={`/${collective.slug}/crypto`}
```

Comment:
Not sure about this. Rationale?

Reply:
@znarf : My thinking here was that we have `donate` route; and `crypto` to me falls under that category; it's a special kinda tier just like the `Custom Contribution`. It has a special card similar to the `Custom Contribution` card. So I thought it makes sense to be consistent with the routes and make it similar to the `donate` route. The `contribute` routes seems only for specific contribution tiers that the user creates; not the predefined ones such as the `Custom Contribution` or `Crypto` in this case. Those were my thoughts; but I am open to changing this if the team thinks otherwise. 😺

Reply:
`donate` and `contribute` are actions, `crypto` is not. So it feels not proper as URL architecture.

Reply:
Makes sense, I've made the changed. 👍🏽

Reply:
@znarf : Actually thinking about this further I changed this to `/payment/crypto` instead of `/contribute/crypto`. It seems that we will have route conflicts if we use the `contribute` verb. It's also mentioned here; 

https://github.com/opencollective/opencollective-frontend/blob/0247cad01dc7a84034265d7f3ae7d65d1ce6fea1/components/contribution-flow/index.js#L434

Let me know your thoughts or if you have a better suggestion for the naming of the verb.